core/integration/features/settlement/0019-MCAL_018.feature

```diff
@@ -88,9 +88,14 @@ Feature: check when settlement data precision is different/equal to the settleme
       | party3 | USD   | ETH/DEC19 | 132000  | 99866000  |
       | lp1    | USD   | ETH/DEC19 | 6600000 | 492200000 |
 
+    When time is updated to "2021-02-11T16:35:24Z"
```

Comment:
Can we not use blocks for this (`When the network moves ahead "N" blocks`). It's a lot easier to read through a scenario using relative time/blocks compared to some arbitrary timestamp and date-time strings. It's also more representative for what actually happens in reality (moving the network by N blocks updates the time 1 second at a time, calling the `OnTick` callbacks each time).

Reply:
Not really, the test moves time by 6 months, so doing it via N blocks would be inefficient.

Reply:
you can set the block duration to 86400 (24h) and move by 180 blocks, or 864000 and move 18 blocks to move time by 6 months

Reply:
Great, there are intermediate steps that move 5 blocks ahead. 

I don't understand why we have "time is updated to" step available at all then if it's such a problem.